### PR TITLE
CASMPET-6118: Update LDAP information for keycloak

### PR DESCRIPTION
**Pull request by ndavidson-hpe**
# Description

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by ndavidson-hpe** [comment]
Fails in style check for two separate notes with only one line between. I want the space in the notes.

**Review by kburns-hpe — APPROVED** [review]

**Review by bklei — APPROVED** [review]

**Comment by johren-hpe** [comment]
A couple of other things I found that should be fixed in Add_LDAP_User_Federation.md:

1.  References to dtr.dev.cray.com should be artifactory.algol60.net/csm-docker/stable/docker.io

2.  May want to change "emailAddress=dcops@hpe.com,CN=Data Center,OU=HPC/MCS,O=HPE,ST=WI,C=US" to match what is now in the new certificate.  Similar change to what Craig made in prepare-site-init.md.

**Comment by rustydb** [comment]
> Fails in style check for two separate notes with only one line between. I want the space in the notes.

Does it fail if you use two new lines?

**Comment by ndavidson-hpe** [comment]
> > Fails in style check for two separate notes with only one line between. I want the space in the notes.
> 
> Does it fail if you use two new lines?

No it fails further with more errors

File: install/prepare_site_init.md [hunk 1]
```diff
@@ -142,6 +142,11 @@ with system-specific customizations.
 
     > **`IMPORTANT`** The CA may not be modified after install.
 
+#### Setup LDAP configuration
```
Comment (rustydb) [inline install/prepare_site_init.md:145]: Just add the "Setup LDAP configuration" and "End of LDAP configuration" to the ToC at the top, nested under the parent group.

**Review by rustydb — CHANGES_REQUESTED** [review]

File: install/prepare_site_init.md (hunk 1 above)
Comment (ndavidson-hpe) [reply install/prepare_site_init.md:145]: Added.

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**